JotunnLib/Managers/LocalizationManager.cs

```diff
@@ -211,9 +213,9 @@ public void AddToken(string token, string value, string language = DefaultLangua
         {
             Dictionary<string, string> languageDict = null;
 
-            if (token.Any(x => Localization.instance.m_endChars.Contains(x)))
+            if (token.Any(x => LocalizationEndChars.Contains(x)))
             {
-                throw new Exception($"Token has an end char in it ({Localization.instance.m_endChars}).");
+                throw new Exception($"Token has an end char in it ({LocalizationEndChars}).");
```

Comment:
that error message :D of course it has. is it is not null it has an end char. but it might not be a faulty one